### PR TITLE
[Word] (typos) Clean up repo readme and some snippets

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/word/40-tables/table-cell-access.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ script:
 
         async function insertTable() {
           await Word.run(async (context) => {
-            // We need a 2D array to hold the initial table values
+            // We need a 2D array to hold the initial table values.
```
Comment (mscharlock) [inline samples/word/40-tables/table-cell-access.yaml]: Might want to spell out two dimensional?

File: snippet-extractor-output/snippets.yaml [hunk 2]
```diff
@@ -3144,17 +3144,17 @@ Word.Paragraph.insertText:
...
         let paragraphs = context.document.body.paragraphs;
-        paragraphs.load("$none"); //Don't need any properties, we'll just wrap each paragraph with a content control
+        paragraphs.load("$none"); // Don't need any properties; we'll just wrap each paragraph with a content control.
 
         await context.sync();
 
         for (let i = 0; i < paragraphs.items.length; i++) {
             let contentControl = paragraphs.items[i].insertContentControl();
-            // For even we tag "even"
+            // For even we tag "even".
```
Comment (mscharlock) [inline snippet-extractor-output/snippets.yaml]:
```suggestion
            // For even, we tag "even".
```

File: samples/word/10-content-controls/insert-and-change-content-controls.yaml [hunk 3]
```diff
@@ -14,16 +14,16 @@ script:
 
 
         async function insertContentControls() {
-            // Traverses each paragraph of the document and wraps a content control on each with either a even or odd tags
+            // Traverses each paragraph of the document and wraps a content control on each with either a even or odd tags.
             await Word.run(async (context) => {
                 let paragraphs = context.document.body.paragraphs;
-                paragraphs.load("$none"); //Don't need any properties, we'll just wrap each paragraph with a content control
+                paragraphs.load("$none"); // Don't need any properties; we'll just wrap each paragraph with a content control.
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply samples/word/10-content-controls/insert-and-change-content-controls.yaml]: I've seen "let's"...

File: snippet-extractor-output/snippets.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply snippet-extractor-output/snippets.yaml]: This file is auto-genned so I've made the update in the snippet itself.

**Pull request merged**